app/src/main/java/in/avimarine/boatangels/BoatHolder.java

```diff
@@ -17,12 +17,14 @@
 
 public class BoatHolder extends RecyclerView.ViewHolder {
   private final TextView mNameField;
+  private final TextView mSumOfferPoints;
```

Comment:
The letter m in the beginning of the variable's name is useless.
This is the style guide we use: https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names
In particular: 
In Google Style special prefixes or suffixes, like those seen in the examples name_, mName, s_name and kName, are **not** used.